stack/core/src/main/java/org/apache/usergrid/corepersistence/CpSetup.java

```diff
@@ -146,17 +145,17 @@ public void setupSystemKeyspace() throws Exception {
 
         migrate();
 
-        cass.createColumnFamily( SYSTEM_KEYSPACE,
-            createColumnFamilyDefinition( SYSTEM_KEYSPACE, APPLICATIONS_CF, ComparatorType.BYTESTYPE ) );
+        cass.createColumnFamily( getApplicationKeyspace(),
```

Comment:
A lot of these CF's are necessary anymore.  Can you audit their usage and delete the ones that are no longer used?  IIRC the only ones we need to retain are the TOKENS_CF and the PRINCIPAL_TOKEN_CF for the inverse relationship